lightning/src/chain/package.rs

```diff
@@ -576,13 +600,22 @@ impl PackageTemplate {
 		}
 		self.height_timer = cmp::min(self.height_timer, merge_from.height_timer);
 	}
-	pub(crate) fn package_amount(&self) -> u64 {
+	/// Gets the amount of all outptus being spent by this package, only valid for malleable
```

Comment:
nit: *outputs an "only valid for now (legacy channel types)"

Reply:
Hmm, not sure what you're suggesting? I'll let you update the docs for anchor in the anchor PRs.